lib/measured/measurable.rb

```diff
@@ -59,6 +59,10 @@ def units
       conversion.unit_names
     end
 
+    def valid_unit?(unit)
+      conversion.unit_or_alias?(unit)
```

Comment:
Probably never possible to have zero or blank units, but maybe do a `.present?` here just for science?